packages/l10n/src/create-l10n-injector/create-l10n-injector.tsx

```diff
@@ -97,6 +92,7 @@ export function createL10NInjector<LoadedData extends {}>({
       const L10NComponent = (props: Props) => {
         const state = useL10n(mapPropsToLocale(props));
         return (
+          // @ts-ignore Related to https://github.com/emotion-js/emotion/issues/3245
```

Comment:
With the latest changes in `main`, perhaps this ignore directive is not needed anymore. Could you please double check? 🙏